synapse_data_warehouse/synapse/dynamic_tables/V2.26.5__remove_ghost_assets_node_latest.sql

```diff
@@ -0,0 +1,26 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+CREATE OR REPLACE DYNAMIC TABLE NODE_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+        WITH latest_unique_rows AS (
+            SELECT
+                *
+            FROM
+                {{database_name}}.synapse_raw.nodesnapshots --noqa: TMP
+            WHERE
+                SNAPSHOT_TIMESTAMP >= CURRENT_TIMESTAMP - INTERVAL '30 DAYS'
```

Comment:
Do we need to increase the time window from 14 to 30 days? Looking at the node you provided as an example in PLFM-8747:

<img width="1048" alt="image" src="https://github.com/user-attachments/assets/c77f6459-4532-4abe-b6b3-438f4867d907" />

While the node is in the trash can it's still being snapshotted as part of the weekly snapshots. So it's sufficient to look at the regular 14-day window and filter where benefactor/parent is the trash can.

Reply:
There are some nodes that still exist on Synapse but have snapshots that are outdated by ~4 weeks (for example [26262846](https://www.synapse.org/Synapse:syn26262846)) and so they're only captured by the query when the window is 30 days:

<img width="298" alt="image" src="https://github.com/user-attachments/assets/d9425e91-31f3-4148-a982-d6b4e50b1c34" />

I haven't yet explored growing the window beyond 30 days, to 40+. I'll run some queries to see if we are missing enough entities that it makes expanding the window worth it (right now the query runs for 6+ mins when the window is 30 days)

Reply:
I see, so this PR is not just trying to address the deleted nodes issue, but also mitigate this issue:
> entities that still exist but for some reason they haven't been snapshotted, and entities that were deleted but it's not represented on Snowflake 

6+ minutes is not too intensive, so I think the current implementation is fine. Although, considering that this affects just ~0.1% of nodes (if I understand your "testing" section correctly) and we can expect an upstream fix (or, at least, find out the true scope/cause of the issue) from platform, we don't need to be too fussy about the time window at this point.

Reply:
Sounds good to me- I ran some more tests and expanded the window up to 60 days. We get a dozen or so new rows every time and yes most of the time these nodes have already been deleted. Let's go with the 30 day window for now and we can iterate on this based on what we hear back from the platform team.